e2e/test/scenarios/embedding/embedding-reproductions.cy.spec.js

```diff
@@ -983,3 +983,39 @@ describe("issue 40660", () => {
     });
   });
 });
+
+describe("issue 47617", () => {
+  const questionDetails = {
+    name: "issue 47617",
+    native: {
+      query: 'SELECT * FROM VALUES (1, 2), (1, 3) AS X("Month", "Value")',
+    },
+    display: "bar",
+    visualization_settings: {
+      "graph.dimensions": ["Month"],
+      "graph.metrics": ["Value"],
+    },
+    enable_embedding: true,
+  };
+
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+
+    createNativeQuestion(questionDetails, { wrapId: true });
+  });
+
+  it("static embed question should render with warnings (metabase#47617)", () => {
+    visitQuestion("@questionId");
+
+    openStaticEmbeddingModal({
+      activeTab: "parameters",
+      previewMode: "preview",
+    });
+
+    getIframeBody().within(() => {
+      cy.findByText("Month").should("be.visible");
+      cy.findByText("Value").should("be.visible");
+    });
+  });
```

Comment:
Does this test assert whether the question has warnings? I see that it finds the "Month" and "Value", which is the graph's labels, but not the warnings.

Reply:
No, the main thing is that the question is rendered. Previously questions with warning will fail to render in static embed/public embed entirely, so this test will fail without this fix which means this tests already assert some behavior (render failure).

Question warning is additional information, it was the root cause of the original issue, but I don't see the need to assert it, I mean we could, but this test already makes sure the question won't crash, so I don't know if an extra assertion would make things any better.

Reply:
@WiNloSt I see, the "should render with warnings" name gave me the wrong impression. We could rename the test to "static embed question with warnings should render" to clarify the meanings.

Reply:
you're right. The test name is kinda vague. I'll update it. Thanks!

Reply:
Fixed in cae9618188c11041566d1107785526bcb8bae53d